src/components/timeline/timeline-chart.js

```diff
@@ -145,7 +130,18 @@ export const TimelineChart = ({ deployments, bounds, campaignName }) => {
 
   return (
     <Disclosure open={!!selectedDeployment}>
-      {geojson?.features?.length && (
+      {geojsonError && (
+        <MapErrorMsg>
+          <ExclamationIcon color={colors.darkTheme.highlight} />
+          <h4>It was not possible to load the map data for this campaign.</h4>
```

Comment:
```suggestion
          <h4>There was an issue loading the map data for this campaign</h4>
```

Reply:
@willemarcel in the case you've suggested that no errors beyond missing data are present, I would suggest we don't call this an error and use a different icon. 

Lets state "Flight path data is not yet available for this campaign" if that is true.